docs-source/content/faq/namespace-management.md

```diff
@@ -5,23 +5,23 @@ draft: false
 weight: 1
 ---
 
-Each WebLogic operator deployment manages a number of Kubernetes namespaces. For information about setting domain namespaces, see [Operator Helm configuration values]({{< relref "/userguide/managing-operators/using-the-operator/using-helm#operator-helm-configuration-values" >}}). A number of Kubernetes resources
+Each operator deployment manages a number of Kubernetes namespaces. For information about setting domain namespaces, see [Operator Helm configuration values]({{< relref "/userguide/managing-operators/using-the-operator/using-helm#operator-helm-configuration-values" >}}). A number of Kubernetes resources
```

Comment:
"setting domain namespaces" feels awkward

Reply:
probably should add a note to say that there can be multiple operators in a k8s cluster, and in that case, care must be taken to ensure their respective lists of domainNamespaces do not overlap.